packages/composables/src/useSyncWishlist.ts

```diff
@@ -27,10 +30,11 @@ export type UseSyncWishlistReturn = {
   /**
    * Wishlist items count
    */
-  count: ComputedRef<number>;
+  totalWishlistItemsCount: Ref<number>;
```

Comment:
I'm afraid that is an unnecessary breaking change; why do you feel the need to change it?

Also it should be a computed property